ui/tests/helpers/policy-generator.js

```diff
@@ -0,0 +1,58 @@
+import { singularize } from 'ember-inflector';
```

Comment:
These policies should be moved to a specific file under `policy-generator` (if we go with my previous suggestion)

Reply:
oops! this file should be deleted, it's a duplicate of the one above. good catch!